lib/ts/controllers/s-expandable-control.ts

```diff
@@ -84,7 +84,8 @@
             if (!attr) {
                 throw "couldn't find controls"
             }
-            const result = document.getElementById(attr);
+            const controlledElements = `#${attr.replace(/ /g, ", #")}`;
+            const result = Array.from(document.querySelectorAll(controlledElements));
```

Comment:
https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#typemapping leads to https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#space-separated-tokens which leads to https://infra.spec.whatwg.org/#ascii-whitespace

> **ASCII whitespace** is U+0009 TAB, U+000A LF, U+000C FF, U+000D CR, or U+0020 SPACE.

So, we need to protect against:

```
aria-controls="   foo
    bar"
```

`attr.split(/\s+/g).filter(s => s.length)` can get a list of ids.

We should also be careful because not everything that is a valid id makes for a valid query selector.  E.g., `aria-controlls="foo.bar"`

The following should give the same results.

```suggestion
            const result = attr.split(/\s+/g).map(s => document.getElementById(s)).filter(e => e);
```